packages/ui/src/components/ProfileItem.tsx

```diff
@@ -0,0 +1,61 @@
+import { HTMLAttributes } from "react";
+
+import { cn } from "@ui/lib/utils";
+
+function ProfileItem({ className, children, ...props }: HTMLAttributes<HTMLDivElement>) {
+  return (
+    <div
+      className={cn(
+        `bg-background-primary relative flex min-h-[2.813rem] min-w-[22.375rem] items-center`,
+        className,
+      )}
+      {...props}
+    >
+      {children}
+    </div>
+  );
+}
+
+function ProfileItemIcon({ children, className, ...props }: HTMLAttributes<HTMLDivElement>) {
+  return (
+    <div
+      className={cn(
+        `text-text-primary mr-[0.938rem] aspect-square min-h-[1.563rem] min-w-[1.563rem]`,
+        className,
+      )}
+      {...props}
+    >
+      {children}
+    </div>
+  );
+}
+
```

Comment:
lucide icon의 아이콘만 children으로 받는 컴포넌트라면 children으로 컴포넌트를 받는 것 보다, lucide icon의 icon을 import하여 타입으로 만들어주고 icon prop을 생성하여 아이콘 네이밍 자동완성을 도와주는 방향으로 구현하는 건 어떨까요?

```suggestion
import { icons } from "lucide-react";

type Icon= keyof typeof icons;
type ProfileItemIconProps = HTMLAttributes<HTMLDivElement> & {
 iconName: Icon;
}

function ProfileItemIcon({ iconName, className, ...props }: ProfileItemIconProps) {
  const Icon = icons[iconName];
  
  return (
    <div
      className={cn(
        `text-text-primary mr-[0.938rem] aspect-square min-h-[1.563rem] min-w-[1.563rem]`,
        className,
      )}
      {...props}
    >
      <Icon />
    </div>
  );
}